ruby/ext/google/protobuf_c/message.c

```diff
@@ -512,7 +512,7 @@ VALUE Message_dup(VALUE _self) {
 }
 
 // Internal only; used by Google::Protobuf.deep_copy.
```

Comment:
If we are exposing this method publicly we should remove this comment.

In many cases I would prefer to keep internal-only methods private. But in this case I don't see any benefit in hiding `Google::Protobuf::Map::deep_copy` while `Google::Protobuf::deep_copy` is public.

Reply:
👍 I'll remove the comments

Reply:
I changed them to be `:nodoc:`.  We usually `:nodoc:` "publicly visible but private" methods in Rails, and I think it makes sense in this case too.